docker-entrypoint-funcs.sh

```diff
@@ -115,12 +115,12 @@ edbdocker_parse_args() {
         EDGEDB_SERVER_BOOTSTRAP_COMMAND="${1#*=}"
         shift
         ;;
-      --bootstrap-script)
-        _edbdocker_parse_arg "EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE" "$1" "$2"
+      --bootstrap-command-file)
```

Comment:
Old spellings must still be supported here.

Reply:
changed